RxSession.podspec

```diff
@@ -36,12 +36,12 @@ Rx Session to use with Endpoints. That's it...
 
   s.dependency 'RxSwift'
   s.dependency 'RxCocoa'
-  s.dependency 'Endpoints'
+  s.dependency 'Endpoints', '~> 2.0.0'
```

Comment:
The pinning should be `~> 2.0` instead of `~> 2.0.0` in my opinion.  
With the first version you get minor + patch bumps, with the second version you only get patch bumps.  
Using semantic versioning, a minor version bump may not break anything.

Reply:
true